.clang-tidy

```diff
@@ -16,6 +16,9 @@ Checks: [
 
   '-android-*',
 
+  '-boost-use-ranges',
```

Comment:
Can't wait to get these two warnings disabled, they are currently spamming my local clang build like hell.